pkg/cmd/cli/describe/describer.go

```diff
@@ -534,6 +534,24 @@ func (d *ImageDescriber) Describe(namespace, name string, settings kctl.Describe
 	return describeImage(image, "")
 }
 
+func describeImageSignature(s imageapi.ImageSignature, out *tabwriter.Writer) error {
+	formatString(out, "\tName", s.Name)
+	formatString(out, "\tType", s.Type)
+	if s.IssuedBy == nil {
+		// FIXME: Make this constant
+		formatString(out, "\tStatus", "Unverified")
```

Comment:
No, it should be either "verified" or "unverified". Any image that was not verified manually is "unverified"... "<none>" might be misleading.